src/zope/server/http/publisherhttpserver.py

```diff
@@ -38,15 +40,17 @@ def __init__(self, request_factory, sub_protocol=None, *args, **kw):
     def _make_application(cls, request_factory, publish=_publish):
         return _make_application(request_factory, publish)
 
+
 def _pmdb_publish(request):
     try:
         return _publish(request, handle_errors=False)
-    except: # pylint:disable=bare-except
+    except:  # noqa: E722 do not use bare 'except'
```

Comment:
This will make @jamadden sad.

Reply:
It does make me sad. This should also work, without losing anything:
```suggestion
    except:  # noqa: E722 pylint:disable=bare-except
```

Reply:
is `except Exception:` an option or is this code expect to catch exceptions that do not inherit from Exception?

Reply:
Well this is the exact reason why a bare `except:` is not considered good... you do not know what to expect ;)